internal/scm/bitbucketserver/bitbucket_server.go

```diff
@@ -7,6 +7,7 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"log"
```

Comment:
Please use the same logging as the rest of the project (`log "github.com/sirupsen/logrus"`)